docs/nginx.md

```diff
@@ -0,0 +1,108 @@
+# Creating Docker NGINX environment
+
+This is a simple example of how to create a Docker environment with NGINX.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/install/)
+
+## Running
+
+Fetch the NGINX image from Docker Hub:
+
+```bash
+docker pull nginx
+```
+
+Set the port to run the container:
+
+```bash
+PORT=7000
+```
+
+Run the container:
+
+```bash
+docker run --name nginx -p $PORT:80 nginx
+```
+
+## Testing
+
+Methods
+
+### Using curl
+
+GET request:
+
+```bash
+curl localhost:$PORT
+```
+
+POST request:
+
+```bash
+curl -X POST localhost:$PORT
+```
+
+### Using browser
+
+Open the browser and type `localhost:$PORT` in the address bar.
+
+## Stopping
+
+Stop the container:
+
+```bash
+docker stop nginx
+```
+
+Remove the container:
+
+```bash
+docker rm nginx
+```
+
+## Updating NGINX configuration to allow POST requests
+
+Edit the `nginx.conf` file:
+
+```bash
+docker exec -it nginx bash
+```
+
+```bash
+vi /etc/nginx/nginx.conf
+```
+
+Add the following lines to the `http` section:
+
+```bash
+http {
```

Comment:
I can't run a POST even if I use this config. It worked with you?

Reply:
It did not work for me neither